v4/source/posts.yaml

```diff
@@ -442,3 +442,35 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  '/posts/{post_id}/actions/{action_id}':
+    post:
+      tags:
+        - posts
+      summary: Perform an post action
```

Comment:
Grammar typo, should be `Perform a post action`

Reply:
Fixed. :)